org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -174,8 +174,6 @@ class Object {
 	 * String representation of Wollok object
 	 */
 	method toString() {
-		// TODO: should be a set
-		//return self.toSmartString(#{})
```

Comment:
Why is it not a Set?

Reply:
Several months ago I tried to uncomment this line, but I experienced performance issues (tests were very slow, maybe the problem is in Set implementation itself)